testsuite/testcases/src/continuous_progress_test.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::NetworkLoadTest;
+use forge::test_utils::consensus_utils::{no_failure_injection, test_consensus_fault_tolerance};
+use forge::{NetworkContext, NetworkTest, Result, Swarm, Test};
+use std::time::Duration;
+use tokio::runtime::Runtime;
+
+pub struct ContinuousProgressTest {
+    pub target_tps: usize,
+}
+
+impl Test for ContinuousProgressTest {
+    fn name(&self) -> &'static str {
+        "performance benchmark"
```

Comment:
We probably want a different name for this test?

Reply:
what is this name used for? we have another name for the suite, which is what I assume is seen elsewhere?

Reply:
Yes, this name is used just in the logs and reports generated by Forge.